### PR TITLE
Amazon Q 3 scopes to 5 scopes migration

### PR DESCRIPTION
**Pull request by hayemaxi**
Continuation of https://github.com/aws/aws-toolkit-vscode/pull/5026

Changes:
- Force the flow by focusing the amazon Q panel specifically for users with a 3 scope connection.
- Additional commit to turn the status bar item red when not connected.

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/core/src/codewhisperer/commands/basicCommands.ts [hunk 1]
```diff
@@ -270,6 +271,13 @@ export const notifyNewCustomizationsCmd = Commands.declare(
     }
 )
 
+function focusQAfterDelay() {
+    // this command won't work without a small delay after install
+    setTimeout(() => {
```
Comment (nkomonen-amazon) [inline packages/core/src/codewhisperer/commands/basicCommands.ts:276]: Use `globals.clock.setTimeout()` instead. All time related things should be done with `globals.clock`. This allows us to stub `globals.clock` in our tests and be able to control the time there.

File: packages/core/src/codewhisperer/service/inlineCompletionService.ts [hunk 2]
```diff
@@ -248,6 +248,7 @@ export class CodeWhispererStatusBar {
             }
             case 'notConnected':
                 statusBar.text = codicon` ${getIcon('vscode-chrome-close')} ${title}`
+                statusBar.backgroundColor = new vscode.ThemeColor('statusBarItem.errorBackground')
```
Comment (nkomonen-amazon) [inline packages/core/src/codewhisperer/service/inlineCompletionService.ts:251]: I'm assuming `errorBackground` is red. Do we really want to make it red when they haven't added a connection yet? I don't think we do this for the Toolkit extension, only showing red once expired.

Comment (hayemaxi) [reply packages/core/src/codewhisperer/service/inlineCompletionService.ts:251]: Yeah this is intended

File: packages/core/src/codewhisperer/commands/basicCommands.ts (hunk 1 above)
Comment (hayemaxi) [reply packages/core/src/codewhisperer/commands/basicCommands.ts]: done

**Review by nkomonen-amazon — APPROVED** [review]

**Pull request merged**